src/Components/Input/Input.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```

Comment:
Rename this file to `Input/Input.js`. This way you can do the import as `../components/Input` instead of `../components/Input/Input`

Do the same in all components where it applies.

Reply:
As mentioned, I renamed files that use the same name as there container's folder.
The new name is 'index.js'.